package.json

```diff
@@ -48,11 +46,7 @@
         "postcss": "^8.3.6",
```

Comment:
Could you pin this to a specific version?

Reply:
What version should be used?